packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -394,7 +394,7 @@ export function TokenImportDialog({
             </div>
           )}
 
-          <div className="mt-6 flex w-full justify-start gap-1 rounded-lg bg-cyan p-3 text-sm text-dark">
+          <div className="mt-6 flex w-full justify-start gap-1 rounded-lg bg-cyan p-3 text-sm text-dark mx-2">
```

Comment:
No change in this file either as it is not a main-screen notification.

Reply:
sure, i will make the changes.